06_modules_lab/02.py

```diff
@@ -1,4 +1,11 @@
 """ Write a program that reads 2 numbers from sys.argv
 and prints their sum.
 """
+import sys
+
+if len(sys.argv)==3:
+    num1 = sys.argv[1]
+    num2 = sys.argv[2]
+    print "Sum of %s + %s = %d" % (num1,num2,int(num1)+int(num2))
+
```

Comment:
Why don't you print a nice "usage" message to the user so she knows how to use your script?